library/asb_get_state.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: asb_get_state
+short_description: Retrieve state from a ConfigMap
+description:
+     - Provides a secure method for allowing Service Bundle developers
+       to access previously set state during a Service Bundle action.
+author:
+    - "Red Hat, Inc."
+options:
+    key:
+        description:
+            - Key of the value that is required
+        required: true
+ env:
```

Comment:
do we want to update the doc here with the new env var? Otherwise I am happy with this now

Reply:
Done